codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowControllerTest.java

```diff
@@ -1050,6 +1050,10 @@ public void error(Throwable t) {
         }
 
         @Override
+        public void writeComplete() {
```

Comment:
I guess we should test that a RST is not sent after a EOS...we went through some heart-ache to address this.